string path
geometry_msgs/Pose capture_pose
---
bool success